lib/logstash/codecs/protobuf.rb

```diff
@@ -0,0 +1,189 @@
+# encoding: utf-8
+require 'logstash/codecs/base'
+require 'logstash/util/charset'
+require 'protocol_buffers' # https://github.com/codekitchen/ruby-protocol-buffers
+
+class LogStash::Codecs::Protobuf < LogStash::Codecs::Base
```

Comment:
Hey guys, sorry for the late reply, summer time and all that. I have added some documentation and I think it's a good start. If something's missing in the documention I will come up with a PR later because right now I'm not sure what else to add. Can we maybe give it a try with this? Thanks a lot :)